packages/grid/backend/grid/api/settings/settings.py

```diff
@@ -14,39 +14,40 @@
 from syft.core.node.common.action.exception_action import ExceptionMessage
 
 # syft
-from syft.core.node.common.node_service.node_setup.node_setup_service import (
-    CreateInitialSetUpMessage,
+from syft.core.node.common.node_service.node_setup.node_setup_messages import (
+    UpdateSetupMessage,
 )
 from syft.core.node.common.node_service.node_setup.node_setup_service import (
     GetSetUpMessage,
 )
 
 # grid absolute
 from grid.api.dependencies.current_user import get_current_user
+from grid.api.users.models import UserPrivate
 from grid.core.node import node
 
 router = APIRouter()
 
 
 @router.post("", status_code=200, response_class=JSONResponse)
-def create_setup(
-    name: str = Body(..., example="Jane Doe"),
-    email: str = Body(..., example="info@openmined.org"),
-    password: str = Body(..., example="changethis"),
+def update_settings(
+    contact: str = Body(..., example="contact@openmined.org"),
     domain_name: str = Body(..., example="OpenGrid"),
+    description: str = Body(..., example="A Domain Node Description ..."),
+    daa: Optional[bool] = False,
```

Comment:
@IonesioJunior what about the `daa_document` field?

Reply:
Fixed by f01698ba2d903fee0009acc176a9d267105384d8